### DISCUSSION TITLE
Where to set flags?

### DISCUSSION THREAD

**Discussion by Gunterlie** [vladmandic/sdnext#275] [Q&A]
I used the Original Auto1111 for a long time where it is easy, but i want to set --flags to run api and such.
Where can these be set?

**Comment by vladmandic** [comment]
api is always-on.
some flags still exist and you can set them the same was as in auto.
but a lot of flags are removed and moved to ui settings.
no functionality that was available with flags is gone.

**Comment by econundrum** [comment]
For the flags that are still their and not moved to settings you can just create a shortcut to webui.bat if you are using windows and add the flags after the name of the file to run e.g. Webui. Bat in the shortcut.

You can make a few shortcuts for different configurations of flags you will commonly want to use.

**Reply by 779575267** [reply]
webui.bat--skip-git  ，am I right?

**Comment by WanderingMindTravels** [comment]
Sorry, I'm still not understanding where to set flags. I saw a comment on another thread about creating (or copying) web-user.bat file from Auto1111. I tried that and also copied the web-user.sh file as well. When I completely restarted, it didn't seem like anything changed. Specifically, I'm running into memory issues when using hi-res fix and would like to set the the --medvram flag. I see it referenced in the documentation, but not sure where to set it. I would also like to use the dark version of a theme.

**Comment by tkiikt** [comment]
@WanderingMindTravels you can create the webui-user.bat with `@webui --medvram`.

**Reply by WanderingMindTravels** [reply]
Thank you for the reply. I only have a rudimentary knowledge of Python. Where do you put that?

**Reply by tkiikt** [reply]
As with A1111, simply put it in the vlad automatic root folder and double-click webui-user.bat.

**Reply by 779575267** [reply]
launch.py: error: unrecognized arguments: --skip-git

**Reply by tkiikt** [reply]
We have no information about your environment or anything else, so there is no way to get beyond that. see this topic: #324

**Comment by PowerfulShell** [comment]


**Reply by vladmandic** [reply]
that still works here. i really don't understand why are you making a big deal out of something that isn't.
the whole point is there are many many ways to do the same thing and its users choice.

> launch.py: error: unrecognized arguments: --skip-git

are you even doing that in root folder of THIS repo (and not changing folder to A1111 or something like that)?
and is repo up-to-date? that flag was added about a week ago.

**Comment by RayStarling** [comment]
edit the bat file to "set COMMANDLINE_ARGS=--api"
